test/tandem/testTandemSimulator.js

```diff
@@ -0,0 +1,608 @@
+      });
+
+      it('throws an error without a status', function() {
+        var fn = function() { simulator.changeReservoir(val); };
+        expect(fn).to.throw(Error);
+      });
+    });
+
+    describe('status', function() {
+      var suspend = {
+        time: '2014-09-25T01:00:00.000Z',
+        deviceTime: '2014-09-25T01:00:00',
+        timezoneOffset: 0,
+        conversionOffset: 0,
+        deviceId: 'InsOmn1234',
+        type: 'deviceEvent',
+        subType: 'status',
+        status: 'suspended',
+        reason: {suspended: 'automatic'}
+      };
+      var resume = builder.makeDeviceEventResume()
+        .with_time('2014-09-25T02:00:00.000Z')
+        .with_deviceTime('2014-09-25T02:00:00')
+        .with_timezoneOffset(0)
+        .with_conversionOffset(0)
+        .with_status('resumed')
+        .with_reason({resumed: 'manual'});
+      var expectedResume = _.assign({}, resume);
+      expectedResume = expectedResume.set('previous', suspend).done();
+
+      it('a suspend passes through', function() {
+        simulator.suspend(suspend);
+        expect(simulator.getEvents()).deep.equals([suspend]);
+      });
+
+      it('a resume passes through', function() {
+        simulator.resume(resume);
+        expect(simulator.getEvents()).deep.equals([resume.done()]);
+      });
+
+      it('a resume includes a previous when preceded by a suspend', function() {
+        simulator.suspend(suspend);
+        simulator.resume(resume);
+        expect(simulator.getEvents()).deep.equals([suspend, expectedResume]);
+      });
+
+      it('uses the timestamp of the first suspend if multiple suspends appear before a single resume', function() {
+        var suspend2 = {
+          time: '2014-09-25T01:05:00.000Z',
+          deviceTime: '2014-09-25T01:05:00',
+          timezoneOffset: 0,
+          conversionOffset: 0,
+          deviceId: 'InsOmn1234',
+          type: 'deviceEvent',
+          subType: 'status',
+          status: 'suspended',
+          reason: {suspended: 'automatic'}
+        };
+        simulator.suspend(suspend);
+        simulator.suspend(suspend2);
+        simulator.resume(resume);
+        expect(simulator.getEvents()).deep.equals([suspend, expectedResume]);
+      });
+    });
+    */
+
+    describe('timeChange', function() {
+      var change = {
+        time: '2014-09-25T01:05:00.000Z',
+        deviceTime: '2014-09-25T01:05:00',
+        timezoneOffset: 0,
+        conversionOffset: 0,
+        deviceId: 'InsOmn1234',
+        type: 'deviceEvent',
+        subType: 'timeChange',
+        change: {
+          from: '2014-09-25T01:05:00',
+          to: '2014-09-25T01:00:00',
+          agent: 'manual'
+        }
+      };
+      it('passes through', function() {
+        simulator.changeDeviceTime(change);
+        expect(simulator.getEvents()).deep.equals([change]);
+      });
+    });
+  });
+
+  /* TODO: revisit these when settings are implemented
```

Comment:
Actually, a test here showing that settings currently *don't* pass through seems appropriate to me.